app/views/hyrax/homepage/_featured_works.html.erb

```diff
@@ -13,7 +13,7 @@
     <%= f.submit("Save order", class: 'btn btn-default') %>
   <% end %>
 <% else %>
-  <ol id="featured_works">
+  <table class="table table-striped" id="featured_works">
```

Comment:
Isn't this semantically a list?

Reply:
Sure. There is some inconsistency in that recent uploads uses a table and featured works uses a list, thus they look completely different (currently). But in the new mockup, the layouts match more closely to the recent list, so I was using that as a framework.

Reply:
A table without table headers can be consider an accessibility concern, so I think this is properly a list.

Reply:
Would table headers for "screen reader only" address that concern? I can switch this to a list if not.

Reply:
If you go the table route and visually hide the headers with `sr-only`, I think that would be alright. It just makes me feel a bit uneasy. I understand that you're trying to match the designs as provided to you. Also, could you scope attributes for the rows and columns for the table.

https://webaim.org/techniques/tables/data#th